zerver/lib/markdown/tabbed_sections.py

```diff
@@ -27,11 +27,11 @@
 """.strip()
 
 NAV_LIST_ITEM_TEMPLATE = """
-<li data-language="{data_language}" tabindex="0">{label}</li>
+<li data-language="{data_tab_key}" tabindex="0">{label}</li>
```

Comment:
I'm pretty sure you need to change the `data-language` here too. The fact that you didn't change this makes me worry you're not testing your work -- how are you testing this?